PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs

```diff
@@ -966,6 +966,7 @@ private void AdjustImageDimensions(ImageItem source)
                     source.Source = CropWindow.RotateResult;
                     source.ImageFile = ImageUtil.GetThumbnailFromFullSizeImg(CropWindow.RotateResult);
                     source.FullSizeImageFile = CropWindow.RotateResult;
+                    source.ContextLink = source.ContextLink + "-Rotated";
```

Comment:
does it reflect the correct ContextLink? usually it's equal to `source.FullSizeImageFile` for local image